app/models/heading.rb

```diff
@@ -41,7 +41,8 @@ def display_short_code
   alias_method :heading_display_short_code, :display_short_code
 
   def display_export_code
-    code[0..-3]
+    code_without_subheading = code[0..-3]
```

Comment:
I wonder if we want to add this method to the declarable concern just to avoid a bit of duplication?

Reply:
We could move it to the `GoodsNomenclature` class

Reply:
So I think the headings that will bear measures are declarable headings, only.

Reply:
https://github.com/trade-tariff/trade-tariff-frontend/blob/main/app/models/concerns/declarable.rb

Reply:
I agree, done!